generators/entity-server/templates/src/main/kotlin/package/web/rest/EntityResource.kt.ejs

```diff
@@ -113,7 +119,7 @@ _%>
 private const val ENTITY_NAME = "<%= entityName %>"
 
 /**
- * REST controller for managing [<%= packageName %>.domain.<%= asEntity(entityClass) %>].
+ * REST controller for managing {@link <%= packageName %>.domain.<%= asEntity(entityClass) %>}.
```

Comment:
KDoc uses [url] for links instead of {@link url}

Reply:
Oops I thought it uses `@link` too... but sadly not. Good catch. 

I think we can also use `@see`

Reply:
Yes, `@see` can be used but links are declared using markdown syntax. See [also](https://kotlinlang.org/docs/reference/kotlin-doc.html).

BTW, doing a quick search there are more existing `@code` and `@link` usages that should be fixed. If you have time to fix them it woul be great; otherwise I will create a separate PR.

Reply:
I think creating an issue makes sense